File: Semestru 4/MPP/lab/.net/lab6/restAPI/restAPI/Model/Domain/LoginInfo.cs
namespace restAPI.Model.Domain
{
	public class LoginInfo : Entity<int>
	{
		public string Username { get; set; }
		public string Password { get; set; }

		public LoginInfo(string username, string password)
		{
			this.Username = username;
			this.Password = password;
		}

		public LoginInfo() { }
	}
}

File: Semestru 4/MPP/lab/java/lab3/Model/src/main/java/org/example/domain/LoginInfo.java
package org.example.domain;

public class LoginInfo extends Entity<Integer>{
    private String Username;
    private String Password;
    private static final long serialVersionUID = 1L;

    public LoginInfo(String username, String password) {
        this.Username = username;
        this.Password = password;
    }

    public String GetUsername() {
        return this.Username;
    }

    public String GetPassword() {
        return this.Password;
    }

    @Override
    public String toString() {
        return "LoginInfo [Id= " + this.GetId() + " Username=" + Username + ", Password=" + Password + "]";
    }
}
